### DISCUSSION TITLE
How to add yaml or yml to the top languages ?

### DISCUSSION THREAD

**Discussion by jinkyler** [anuraghazra/github-readme-stats#1070] [Q&A] [answered]
help me, please !!

**Comment by florianbussmann** [comment, marked as answer]
This should already be the case.
In another discussion about adding nim as language @anuraghazra referred to linguist https://github.com/anuraghazra/github-readme-stats/discussions/809#discussioncomment-279223

If you look at the linguist repo you will find that YAML is already detected: https://github.com/github/linguist/blob/19b3a9bd7d585be8d6459cfab0a51607b2aee68b/lib/linguist/languages.yml#L6585

However top languages are based on `how much code in bytes` you have in that language and I would say that yaml files are usually very small compared to the rest of the codebase https://github.com/anuraghazra/github-readme-stats/issues/450#issuecomment-770357145
